### PR TITLE
docs: Add badges to readme

### PR DESCRIPTION
**Pull request by soda480**
Add relevant badges to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
...
....io/github/v/tag/edgexfoundry/device-onvif-camera?include_prereleases&sort=semver&label=latest-dev)](https://github.com/edgexfoundry/device-onvif-camera/tags) ![GitHub Latest Stable Tag)](https://img.shields.io/github/v/tag/edgexfoundry/device-onvif-camera?sort=semver&label=latest-stable) [![GitHub License](https://img.shields.io/github/license/edgexfoundry/device-onvif-camera)](https://choosealicense.com/licenses/apache-2.0/) ![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/edgexfoundry/device-onvif-camera) [![GitHub Pull Requests](https://img.shields.io/github/issues-pr-raw/edgexfoundry/device-onvif-camera)](https://github.com/edgexfoundry/device-onvif-camera/pulls) [![GitHub Contributors](https://img.shields.io/github/contributors/edgexfoundry/device-onvif-camera)](https://github.com/edgexfoundry/device-onvif-camera/contributors) [![GitHub Committers](https://img.shields.io/badge/team-committers-green)](https://github.com/orgs/edgexfoundry/teams/device-onvif-camera-committers/members) [![GitHub Commit Activity](https://img.shields.io/github/commit-activity/m/edgexfoundry/device-onvif-camera)](https://github.com/edgexfoundry/device-onvif-camera/commits)
```
Comment (lenny-goodell) [inline README.md:2]: Committers link returns 404.
Should be `https://github.com/orgs/edgexfoundry/teams/device-onvif-camera-commiters/members`

**Review by lenny-goodell — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (soda480) [reply README.md:2]: for consistency sake the team name should have been named `device-onvif-camera-committers`, I'll submit an LF ticket to get the team renamed.

Comment (lenny-goodell) [reply README.md:2]: Yep, I didn't catch they miss spelled committers... ;-)

**Review by lenny-goodell — APPROVED** [review]
LGTM.

**Review by bill-mahoney — APPROVED** [review]

**Review by ernestojeda — APPROVED** [review]

**Pull request merged**